src/amplitude/amplitude.ts

```diff
@@ -1,9 +1,8 @@
-import {erProd} from '../utils/url-utils';
+//import {erProd} from '../utils/url-utils';
 import {AmplitudeEvent} from './taxonomy-events';
 
 export function initAmplitude() {
-    const projectId = erProd() ? '691963e61d2b11465aa96acfcaa8959b' : 'faf28eb5445abfe75c7ac28ae7a8d050';
-
+    //  const projectId = erProd() ? '691963e61d2b11465aa96acfcaa8959b' : 'faf28eb5445abfe75c7ac28ae7a8d050';
```

Comment:
Hva skjer her?

Reply:
Hehe ja ikke sant, husker du at jeg trodde jeg hadde merget at vi fjernet amplitude og node for den econnreset feilen? gjorde jo ikke det fordi prettier klaget når man skulle ut, men ikke på pr nivå... sååe ja dette er bare for å roe prettier i alle fall hehe :P